### PR TITLE
[Docs] 增加第二类api difference

### PR DESCRIPTION
**Pull request by hd9568**

### CODE REVIEW

**Comment by zhwesky2010** [comment]
infoflow 2025-10-30 15-53-37

infoflow 2025-10-30 15-53-51



这些对齐一下格式吧，这样：

infoflow 2025-10-30 15-59-30

**Comment by zhwesky2010** [comment]
infoflow 2025-10-30 16-03-34

这种错误有办法检查吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.max.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## [ 仅 API 调用方式不一致 ]torch.max
+
+### [torch.max](https://pytorch.org/docs/stable/generated/torch.max.html)
+
+```python
+torch.max(input, *, out=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.max.md]: 这几个签名直接写 `torch.max(*args, **kwargs)` 吧

max、min、median、nanmedian

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.max.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+paddle.compat.max(input, *, out=None)
+```
+
+两者功能一致，但调用方式不一致，具体如下：
+
+### 转写示例
+
+```python
+# PyTorch 写法
+result = torch.max(x)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.max.md:21]: 备注一下：torch.max有多种签名与用法，均只需修改torch前缀为paddle.compat

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.median.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+paddle.compat.median(input, dim=None, keepdim=False, *, out=None)
+```
+
+两者功能一致，但调用方式不一致，具体如下：
+
+### 转写示例
+
+```python
+# PyTorch 写法
+result = torch.median(input)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.median.md:21]: 同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.min.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+torch.min(input, *, out=None)
+```
+
+### [paddle.compat.min](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/min_cn.html#paddle/compat/min_cn#cn-api-paddle-compat-min)
+
+```python
+paddle.compat.min(input, *args, out=None, **kwargs)
+```
+
+两者功能一致，但调用方式不一致，具体如下：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.min.md]: 同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nanmedian.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+paddle.compat.nanmedian(input, dim=None, keepdim=False, *, out=None)
+```
+
+两者功能一致，但调用方式不一致，具体如下：
+
+### 转写示例
+
+```python
+# PyTorch 写法
+result = torch.nanmedian(input)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nanmedian.md:21]: 同上

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.softmax.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+### [torch.nn.functional.softmax](https://pytorch.org/docs/stable/nn.functional.html#torch.nn.functional.softmax)
+
+```python
+torch.nn.functional.softmax(input, dim=None, _stacklevel=3, dtype=None)
+```
+
+### [paddle.compat.softmax](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/softmax_cn.html#paddle/compat/softmax_cn#cn-api-paddle-compat-softmax)
+
+```python
+paddle.compat.softmax(input, dim=None, dtype, *, out=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.softmax.md]: 如果是 [仅API调用方式不一致]，上下的签名，**API参数部分应该都是一样的，或者paddle比torch参数更多**。

这里需要再整体核对下，为什么会有的参数不符合上面的情况。

paddle.compat.softmax的_stacklevel、dtype都是一样的。多一个out。

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.median.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+### [torch.median](https://pytorch.org/docs/stable/generated/torch.median.html)
+
+```python
+torch.median(*args, **kwargs)
+```
+
+### [paddle.compat.median](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/median_cn.html#paddle/compat/median_cn#cn-api-paddle-compat-median)
+
+```python
+paddle.compat.median(input, dim=None, keepdim=False, *, out=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.median.md]: 上面如果按*args, **kwargs写，这里也这样写吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.min.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+### [torch.min](https://pytorch.org/docs/stable/generated/torch.min.html)
+
+```python
+torch.min(*args, **kwargs)
+```
+
+### [paddle.compat.min](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/min_cn.html#paddle/compat/min_cn#cn-api-paddle-compat-min)
+
+```python
+paddle.compat.min(input, *args, out=None, **kwargs)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.min.md]: *args, **kwargs

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nanmedian.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+### [torch.nanmedian](https://pytorch.org/docs/stable/generated/torch.nanmedian.html)
+
+```python
+torch.nanmedian(*args, **kwargs)
+```
+
+### [paddle.compat.nanmedian](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/nanmedian_cn.html#paddle/compat/nanmedian_cn#cn-api-paddle-compat-nanmedian)
+
+```python
+paddle.compat.nanmedian(input, dim=None, keepdim=False, *, out=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nanmedian.md]: *args, **kwargs

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.Unfold.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+两者功能一致，但调用方式不一致，具体如下：
+
+### 转写示例
+
+```python
+# PyTorch 写法
+unfold = torch.nn.Unfold(kernel_size=(2, 2))
+
+# Paddle 写法
+unfold = paddle.compat.Unfold(kernel_size=(2, 2))
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.Unfold.md]: 这个更新成了 `paddle.compat.nn.Unfold`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.Unfold.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+### [torch.nn.Unfold](https://pytorch.org/docs/stable/generated/torch.nn.Unfold.html#torch.nn.Unfold)
+
+```python
+torch.nn.Unfold(kernel_size, dilation=1, padding=0, stride=1)
+```
+
+### [paddle.compat.Unfold](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/Unfold_cn.html#paddle/compat/Unfold_cn#cn-api-paddle-compat-Unfold)
+
+```python
+paddle.compat.Unfold(kernel_size, dilation=1, padding=0, stride=1)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.Unfold.md]: paconvert更新为`paddle.compat.nn.Unfold`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.pad.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+### [torch.nn.functional.pad](https://pytorch.org/docs/stable/nn.functional.html#torch.nn.functional.pad)
+
+```python
+torch.nn.functional.pad(input, pad, mode="constant", value=None)
+```
+
+### [paddle.compat.pad](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/pad_cn.html#paddle/compat/pad_cn#cn-api-paddle-compat-pad)
+
+```python
+paddle.compat.pad(input, pad, mode="constant", value=0.0)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.pad.md]: paconvert更新为`paddle.compat.nn.functional.pad`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.softmax.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+### [torch.nn.functional.softmax](https://pytorch.org/docs/stable/nn.functional.html#torch.nn.functional.softmax)
+
+```python
+torch.nn.functional.softmax(input, dim=None, _stacklevel=3, dtype=None)
+```
+
+### [paddle.compat.softmax](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/softmax_cn.html#paddle/compat/softmax_cn#cn-api-paddle-compat-softmax)
+
+```python
+paddle.compat.softmax(input, dim=None, _stacklevel=3, dtype=None, out=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.softmax.md]: paconvert更新为`paddle.compat.nn.functional.softmax`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.cuda.is_bf16_supported.md [hunk 14]
```diff
@@ -12,14 +12,11 @@ torch.cuda.is_bf16_supported(including_emulation=True)
...
-```python
-# PyTorch 写法
-result = torch.cuda.is_bf16_supported()
-
-# Paddle 写法
-result = paddle.amp.is_bfloat16_supported()
-```
+| PyTorch    | PaddlePaddle | 备注 |
+| ---------- | ------------ | -- |
+| including_emulation | - | 是否包含软件模拟支持 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.cuda.is_bf16_supported.md]: 看一下模板，如果没有转写示例，需要标明 暂无转写方式

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.cuda.is_bf16_supported.md [hunk 15]
```diff
@@ -12,14 +12,11 @@ torch.cuda.is_bf16_supported(including_emulation=True)
...
+| - | device | 查询的设备类型 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.cuda.is_bf16_supported.md]: 看一下模板规范，需要写一下paddle应该怎么设置，不需要修改的话就是 paddle保持默认即可

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.Module.train.md [hunk 16]
```diff
@@ -12,16 +12,10 @@ torch.nn.Module.train(mode=True)
...
-model = torch.nn.Linear(10, 20)
-model.train()
-
-# Paddle 写法
-model = paddle.nn.Linear(10, 20)
-model.train()
-```
+| PyTorch    | PaddlePaddle | 备注 |
+| ---------- | ------------ | -- |
+| `mode`     | 无           | 是否设置为训练模式 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.nn.Module.train.md]: 同上，这里不符合模板规范

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.optim.Optimizer.step.md [hunk 17]
```diff
@@ -12,24 +12,10 @@ torch.optim.Optimizer.step(closure=None)
...
-sgd = paddle.optimizer.SGD(
-     learning_rate=0.1,
-     parameters=linear.parameters(),
-     weight_decay=0.01
-)
-sgd.step()
-```
+| PyTorch    | PaddlePaddle | 备注 |
+| ---------- | ------------ | -- |
+| closure    | -           | 传入一个闭包函数，在优化器更新参数前执行。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.optim.Optimizer.step.md]: 同上，这里不符合模板规范

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.pad.md [hunk 18]
```diff
@@ -6,10 +6,10 @@
 torch.nn.functional.pad(input, pad, mode="constant", value=None)
 ```
 
-### [paddle.compat.pad](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/pad_cn.html#paddle/compat/pad_cn#cn-api-paddle-compat-pad)
+### [paddle.compat.nn.pad](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/nn/pad_cn.html#paddle/compat/nn/pad_cn#cn-api-paddle-compat-nn-pad)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.pad.md]: `paddle.compat.nn.functional.pad`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.cuda.is_bf16_supported.md [hunk 19]
```diff
@@ -18,5 +18,5 @@ PyTorch 相比 Paddle 支持更多其他参数，具体如下：
 
 | PyTorch    | PaddlePaddle | 备注 |
 | ---------- | ------------ | -- |
-| including_emulation | - | 是否包含软件模拟支持 |
-| - | device | 查询的设备类型 |
+| including_emulation | - | 是否包含软件模拟支持，暂无转写方式 |
+| - | device | 查询的设备类型，转写时 Paddle 保持默认 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch_more_args/torch.cuda.is_bf16_supported.md]: Paddle 保持默认即可

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.softmax.md [hunk 20]
```diff
@@ -6,10 +6,10 @@
 torch.nn.functional.softmax(input, dim=None, _stacklevel=3, dtype=None)
 ```
 
-### [paddle.compat.nn.softmax](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/nn/softmax_cn.html#paddle/compat/nn/softmax_cn#cn-api-paddle-compat-nn-softmax)
+### [paddle.compat.nn.functional.softmax](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/compat/nn/functional/softmax_cn.html#paddle/compat/nn/softmax_cn#cn-api-paddle-compat-nn-softmax)
 
 ```python
-paddle.compat.nn.softmax(input, dim=None, _stacklevel=3, dtype=None, out=None)
+paddle.compat.nn.functional.softmax(input, dim=None, _stacklevel=3, dtype=None, out=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.softmax.md:12]: 这些在后面优化 validate_api_difference 及一致性工具时，加上校验吧，需要与paconvert的pytorch api、paddle API保持一致。

**Review by zhwesky2010 — COMMENTED** [review]
LGTM

File: docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.softmax.md (hunk 20 above)
Comment (hd9568) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/invok_only_diff/torch.nn.functional.softmax.md:12]: 收到

**Review by zhwesky2010 — APPROVED** [review]
